### ISSUE TITLE
NZBplanet.net returns insufficient caps call

### ISSUE THREAD

**Issue by PearsonFlyer** [Prowlarr/Prowlarr#279]
NZBPlanet returns a caps call that doesn't say what query types it supports. This results in incorrect (text-only) searches with this indexer.

They return:

<searching>
<search available="yes"/>
<tv-search available="yes"/>
<movie-search available="yes"/>
<audio-search available="yes"/>
</searching>

And they should return something like this:

<searching>
<search available="yes" supportedParams="q,group"/>
<tv-search available="yes" supportedParams="q,rid,tvdbid,season,ep"/>
<movie-search available="yes" supportedParams="q,imdbid,genre"/>
<audio-search available="no" supportedParams="q,album,artist,label,year,genre"/>
</searching>

**To Reproduce**

1)  Do a caps call on Planet directly:

https://nzbplanet.net/api?t=caps&apikey=yourapikey

This matches the results in Prowlarr, so it shows it's an issue with the indexer directly.

I think Planet will either need to fix their caps call, or Prowlarr will need to adjust and pretend like it gave a good result.

[AB#25](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/25)

**Comment by PearsonFlyer** [comment]
Note: I have been in contact with nzbplanet, and they have passed this request on to their admin to review/fix their caps call. We may still want to handle this in Prowlarr for this and any other newznab indexer that has a bad caps call. We should at least assume they can do `q` and possibly a few other things.

**Comment by RobinDadswell** [comment]
> Note: I have been in contact with nzbplanet, and they have passed this request on to their admin to review/fix their caps call. We may still want to handle this in Prowlarr for this and any other newznab indexer that has a bad caps call. We should at least assume they can do `q` and possibly a few other things.

Any updates on this?

**Comment by PearsonFlyer** [comment]
Planet said they're not going to fix it, and they have  veered from newznab compatibility several years ago, and they do not have any interest in changing this as it might break other tools.

**Comment by bakerboy448** [comment]
was that for the caps or categories?

**Comment by PearsonFlyer** [comment]
It's both. It should be reporting WEBDL, category 2080, in the caps, but it's reporting WEB-DL, category 2035 instead. Prowlarr won't recognize that as a valid WEBDL category so it keep it unchecked. The overall Movies (2000) category is checked, so it shouldn't really matter, but it's an annoying miss.

**Comment by bakerboy448** [comment]
FYI link appears to work w/o an API key 

```
<searching>
<search available="yes" supportedParams="q"/>
<tv-search available="yes" supportedParams="q,rid,tvdbid,tvmazeid,season,ep"/>
<movie-search available="yes" supportedParams="q,imdbid,genre"/>
</searching>
```

I believe the searching part is now resolved so we can close this?

**Comment by PearsonFlyer** [comment]
Yep, that part is working now. I'd say it's safe to close this.

**Comment by jonathan-bloodworth** [comment]
Can someone ELI5 to me why nzbplanet works through hydra and directly from radarr/sonarr please? Is this down to the way prowlarr is built that means I'm still unable to see results from planet via prowlarr? Just asking so I can understand this issue a little better.

Thanks

**Comment by bakerboy448** [comment]
Planet works with Prowlarr.

Please use one of the support channels: [Reddit](https://www.reddit.com/r/prowlarr/) or  [Discord](https://prowlarr.com/discord) for support/questions.
